qa/tasks/cephfs/test_admin.py

```diff
@@ -1316,6 +1317,294 @@ def setup_test_env(self, fs_auth_caps):
             self.setup_for_multiple_paths(fs_auth_caps)
 
 
+class TestFsAuthorizeUpdate(CephFSTestCase):
+    client_id = 'testuser'
+    client_name = f'client.{client_id}'
+    CLIENTS_REQUIRED = 2
+    MDSS_REQUIRED = 2
+
+    ######################################
+    # cases where "fs authorize" adds caps
+    ######################################
+
+    def test_add_caps_for_another_FS(self):
+        """
+        Test that "ceph fs authorize" adds caps for a second FS to a keyring
+        that already had caps for first FS.
+        """
+        self.fs1 = self.fs
+        self.fs2 = self.mds_cluster.newfs('testcephfs2')
```

Comment:
This test class has `MDSS_REQUIRED = 2` - were any warnings generated when a new file system was created (since there won't be any standby MDSs left)? If not, then is the warning in ignore-list set of warnings?

Reply:
Any warning generated in teuthology logs? No idea.

Reply:
If the number of MDSs configured in the cluster is 2, then yes, a warning is generated. I think we run this test with >2 MDSs, so its fine, yes?

Reply:
> I think we run this test with >2 MDSs, so its fine, yes?

MDSS_REQUIRED is set to 2, so  this can't be true.